src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, hidden) {
+    super(name);
+    this.hidden = false;
+  }
```

Comment:
```suggestion
  hidden = false;
```
you don't use `hidden` from `constructor` anyway